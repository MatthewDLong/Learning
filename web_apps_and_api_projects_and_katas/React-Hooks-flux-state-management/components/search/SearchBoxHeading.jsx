import React from "react";
import { searchbox__title } from "./search.css";

const SearchBoxHeading = () => (
  <h2 className={searchbox__title}>Where are you going?</h2>
);

export default SearchBoxHeading;
