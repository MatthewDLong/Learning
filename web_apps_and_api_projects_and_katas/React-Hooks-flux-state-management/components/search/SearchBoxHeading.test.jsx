import React from "react";
import SearchBoxHeading from "./SearchBoxHeading";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<SearchBoxHeading />).toJSON();
  expect(tree).toMatchSnapshot();
});
