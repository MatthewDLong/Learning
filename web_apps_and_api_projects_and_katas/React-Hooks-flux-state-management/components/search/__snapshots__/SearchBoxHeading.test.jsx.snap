// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2
  className="searchbox__title"
>
  Where are you going?
</h2>
`;
